.github/workflows/org-quality-gate.yml

```
@@ -0,0 +1,43 @@
+    - cron: '17 5 * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  actions: read
+
+jobs:
+  quality_and_security:
+    name: quality_and_security
+    if: ${{ hashFiles('.github/workflows/code-quality.yml') != '' }}
+    uses: ./.github/workflows/code-quality.yml
```

actionlint (1.7.11)

[error] could not read reusable workflow file for "./.github/workflows/code-quality.yml": open /home/jailuser/git/.github/workflows/code-quality.yml: no such file or directory

(workflow-call)